### PR TITLE
added  6 new recipes

### PR DESCRIPTION
**Pull request by sawood164**
⚠️ THIS IS A PULL REQUEST TEMPLATE! ⚠️ This helps you complete a pull request. Please follow the instructions below. Reminder that you are bound to abide by the [Code of Conduct](../CODE_OF_CONDUCT.md)

## Submission Checklist ✅
Open a pull request only after you have finished adding your recipes. Please complete the below checklist. Failure to follow these instructions may result in your pull request being closed without review, and may affect your contribution.

**Important**: *Reviewers will review your pull requests when applicable, based on a queue. Not starring the repo, will lead to questions*

-   [x] I have starred and forked the repo
-   [x] I read the instructions to submit the page.
-   [x] I have created a new page, inside a folder named `html-files`.
-   [x] I have successfully added 6 new recipes, that are not recurring.
-   [x] I have added a `reviewer` to this pull request.
-   [x] I have added all the images inside a separate page named after my page name.
-   [x] I have uploaded all the images to the images folder, in transparent ```.png``` format and are compressed to under 500kb.
-   [x] I understand that no image links should be used. All the images should be uploaded.
-   [x] I understand that a reviewer will merge my pull request after review, and that I must comply to recieving code reviews.

### CODE REVIEW

**Comment by sawood164** [comment]
added recipes

File: images/Page 16/Karahi.jpg
Comment (TheGuyDangerous) [inline images/Page 16/Karahi.jpg:1]: Convert the img to png , n make it transparent

File: images/Page 16/Haleem.webp
Comment (TheGuyDangerous) [inline images/Page 16/Haleem.webp:1]: img is in .webp format. convert it to .png format instead

File: images/Page 16/Baklava.webp
Comment (TheGuyDangerous) [inline images/Page 16/Baklava.webp:1]: img is in .webp format. convert it to .png format instead

File: images/Page 16/Shawarma.webp
Comment (TheGuyDangerous) [inline images/Page 16/Shawarma.webp:1]: img is in .webp format. convert it to .png format instead

**Review by TheGuyDangerous — CHANGES_REQUESTED** [review]

**Review by TheGuyDangerous — APPROVED** [review]
Well and good

**Pull request merged**